src/test/regress/expected/distributed_types.out

```diff
@@ -425,7 +425,7 @@ $$);
  (localhost,57638,t,"CREATE DOMAIN")
 (2 rows)
 
-CREATE TABLE domain_indirection_test (f1 int, f3 domain);
+CREATE TABLE domain_indirection_test (f1 int, f3 domain, domain_array domain[]);
```

Comment:
For readers, I couldn't add this column with `ALTER TABLE` command due to  https://github.com/citusdata/citus/issues/3678
That is why I changed `CREATE TABLE` command